### DISCUSSION TITLE
Can not use my academic email address, GitHub requires using an old domain that is no longer in use.

### DISCUSSION THREAD

**Discussion by redelmann** [community/community#43747] [GitHub Education]
Hi,

I am a high school teacher at Gymnase de Burier in Switzerland.
I can not register to GitHub Education as the form requires a different email address from a domain no longer in use when I select my school.
How can I fix this ?

Cheers,
Romain Edelmann

**Comment by not-nullptr** [comment]
in the exact same boat here. don't bother emailing support, they'll wait a week then send u a canned response directing you to "support yourself as a student" here.
